src/BuiltInTools/dotnet-watch/HotReload/DefaultDeltaApplier.cs

```diff
@@ -29,14 +30,8 @@ public DefaultDeltaApplier(IReporter reporter)
 
         public bool SuppressBrowserRefreshAfterApply { get; init; }
 
-        public async ValueTask InitializeAsync(DotNetWatchContext context, CancellationToken cancellationToken)
+        public ValueTask InitializeAsync(DotNetWatchContext context, CancellationToken cancellationToken)
         {
-            if (_pipe is not null)
```

Comment:
Why was this removed? It seems like good hygiene to have it?

Reply:
Turns out we were newing up these instances in every hot reload loop. So this was never true. Also why the named pipe name is a static because we were only configuring it once.